### PR TITLE
Add ability to create empty table

### PR DESCRIPTION
**Pull request by turnerm**
Add "make empty" option to makefile, this helps with testing against hapi-pipelines.

@b-j-mills FYI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,3 +40,10 @@ ${TEST_DATABASE}: ${DATABASE_DIR} ${CORE_SCHEMA}
 
 ${DATABASE_DIR}:
 	mkdir -p ${DATABASE_DIR}
+
+# Option to make empty DB
+
+test-database-empty: ${DATABASE_DIR} ${CORE_SCHEMA}
+	cat ${TEST_SCHEMAS} ${TEST_VIEWS} | ${SQLITE_CMD} ${TEST_DATABASE}
+
+empty: test-database-delete test-database-empty
```
Comment (davidmegginson) [inline Makefile:49]: You can remove this target.

File: Makefile [hunk 2]
```diff
@@ -39,3 +40,10 @@ ${TEST_DATABASE}: ${DATABASE_DIR} ${CORE_SCHEMA}
 
 ${DATABASE_DIR}:
 	mkdir -p ${DATABASE_DIR}
+
+# Option to make empty DB
+
+test-database-empty: ${DATABASE_DIR} ${CORE_SCHEMA}
```
Comment (davidmegginson) [inline Makefile:46]: Add test-database-delete as the first dependency for this target; that way, there won't be an error if the test database is already populated.

**Review by davidmegginson — CHANGES_REQUESTED** [review]
I suggested a couple of changes. OK to go ahead and merge either way.

**Comment by davidmegginson** [comment]
Will this need more changes after we merge https://github.com/OCHA-DAP/hapi-schemas/pull/12 ?

**Comment by turnerm** [comment]
Superseded by #16

**Pull request closed without merging**